utils/pdfGenerator.js

```
@@ -1,50 +1,51 @@
+const fs = require('fs');
+
+// Hardcoded secret (Security Risk)
+const SECRET = 'mySuperSecretPassword123';
+
+// Unused function (Duplicate/Unused Code)
+function doNothing() {
+  return 42;
+}
+
+// Duplicate function (Duplicate/Unused Code)
+function doNothing() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'doNothing'. Consider to delete it or rename it.

'doNothing' is defined here:


(lint/suspicious/noRedeclare)